app/controllers/admin/resources_controller.rb

```diff
@@ -23,12 +23,19 @@ def create
     end
 
     def update
-      if @resource.update_attributes(resource_params)
-        redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
-                    notice: 'Resource successfully updated.'
-      else
-        flash[:error] = "Resource update failed: #{@resource.errors.full_messages.join('. ')}."
-        render :edit
+      respond_to do |format|
+        if @resource.update_attributes(resource_params)
+          format.html do
+            redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
+                        notice: 'Resource successfully updated.'
+          end
+          format.js{ render action: 'update_quantity' }
```

Comment:
Added flash message for else part
![error](https://cloud.githubusercontent.com/assets/14261624/23264335/f79753ca-fa06-11e6-8528-1be87e88ffab.png)